server/scripts/20220722000000-backfill-subscriptions.ts

```diff
@@ -0,0 +1,65 @@
+import "./bootstrap";
+import { Subscription, Document, Event } from "@server/models";
+
+const limit = 100;
+let page = parseInt(process.argv[2], 10);
+page = Number.isNaN(page) ? 0 : page;
+
+export default async function main(exit = false) {
+  const work = async (page: number) => {
+    console.log(`Backfill subscription… page ${page}`);
+
+    // Retrieve all documents within set limit.
+    const documents = await Document.findAll({
+      limit,
+      offset: page * limit,
+      order: [["createdAt", "DESC"]],
+    });
+
+    for (const document of documents) {
+      try {
+        const collaborators = document.collaboratorIds;
+
+        for (const collaborator of collaborators) {
+          const [
+            collaboratorSubscription,
+            created,
+          ] = await Subscription.findOrCreate({
+            where: {
+              userId: collaborator,
+              documentId: document.id,
+              event: "documents.update",
+            },
+          });
```

Comment:
```suggestion
        await Promise.all(document.collaboratorIds, (collaboratorId) => 
          Subscription.findOrCreate({
            where: {
              userId: collaboratorId,
              documentId: document.id,
              event: "documents.update",
            },
          })
        })
```

Parallelize creation will speed this script up a lot